Errors after opening the menu

Couldn't include file 'sqlworkbench\ace\ace.js.1.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 520))
Couldn't include file 'sqlworkbench\ace\ace.js.2.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 520))
Couldn't include file 'sqlworkbench\ace\ext-language_tools.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 521))
Couldn't include file 'sqlworkbench\ace\theme-monokai.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 522))
Couldn't include file 'sqlworkbench\ace\mode-mysql.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 523))
Couldn't include file 'sqlworkbench\ace\mode-sql.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 524))
Couldn't include file 'sqlworkbench\ace\snippets\mysql.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 525))
Couldn't include file 'sqlworkbench\ace\snippets\sql.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 526))
Couldn't include file 'sqlworkbench\ace\sql-formatter.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 527))

[ERROR] addons/sqlworkbench/lua/sqlworkbench/menu.lua:592: attempt to concatenate a nil value
  1. CreateConnection - addons/sqlworkbench/lua/sqlworkbench/menu.lua:592
   2. func - addons/sqlworkbench/lua/sqlworkbench/menu.lua:1113
    3. unknown - lua/includes/extensions/net.lua:32

Player [2][Nexeonenn] ERR: Couldn't include file 'sqlworkbench\ace\ace.js.1.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 520))
Player [2][Nexeonenn] ERR: Couldn't include file 'sqlworkbench\ace\ace.js.2.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 520))
Player [2][Nexeonenn] ERR: Couldn't include file 'sqlworkbench\ace\ext-language_tools.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 521))
Player [2][Nexeonenn] ERR: Couldn't include file 'sqlworkbench\ace\theme-monokai.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 522))
Player [2][Nexeonenn] ERR: Couldn't include file 'sqlworkbench\ace\mode-mysql.js.lua' (File not found) (@addons/sqlworkbench/lua/sqlworkbench/menu.lua (line 523))

--- a/lua/autorun/sqlworkbench.lua
+++ b/lua/autorun/sqlworkbench.lua
@@ -65,10 +65,10 @@ if (SERVER) then
 	AddCSLuaFile("sqlworkbench/menu.lua")
 	AddCSLuaFile("sqlworkbench/networking.lua")
 
-	for _,f in ipairs((file.Find("sqlworkbench/ace/*.lua", "LUA"))) do
+	for _,f in ipairs((file.Find("sqlworkbench/ace/*", "LUA"))) do
 		AddCSLuaFile("sqlworkbench/ace/" .. f)
 	end
-	for _,f in ipairs((file.Find("sqlworkbench/ace/snippets/*.lua", "LUA"))) do
+	for _,f in ipairs((file.Find("sqlworkbench/ace/snippets/*", "LUA"))) do
 		AddCSLuaFile("sqlworkbench/ace/snippets/" .. f)
 	end
 

